toolkit/scripts/toolkit.mk

```diff
@@ -13,10 +13,12 @@ toolkit_component_extra_files = \
 	$(toolkit_root)/.gitignore
 
 # Outputs
-toolkit_archive   = $(OUT_DIR)/toolkit-$(RELEASE_VERSION).tar.gz
+toolkit_version   = $(RELEASE_VERSION)-$(build_arch)
```

Comment:
I am generally in favor of this change, but I am concernted that almost every single derivative pipeline will be impacted...as it is, we just went through and fixed quite a few of them.  Please take care that the derivative pipelines can ingest this.